src/m365/spo/commands/site/site-set.spec.ts

```diff
@@ -1436,7 +1436,8 @@ describe(commands.SITE_SET, () => {
       return Promise.reject('Invalid request');
     });
     sinon.stub(request, 'post').callsFake((opts) => {
-      if (opts.url === 'https://contoso-admin.sharepoint.com/_vti_bin/client.svc/ProcessQuery') {
+      if ((opts.url as string).indexOf(`/_vti_bin/client.svc/ProcessQuery`) > -1 &&
```

Comment:
Actually, it was better before the change. As best practice we agreed that for new/refactored unit tests we should check URLs using `===` and not `.indexOf`